### ISSUE TITLE
✨ MerkleTree

### ISSUE THREAD

**Issue by Vectorized** [Vectorized/solady#98]
This one won't be mostly Yul.

**Comment by ameya-deshmukh** [comment]
@Vectorized ser, is this still open to take up?

**Comment by Vectorized** [comment]
Yes. But not in a rush. I think less than 0.01% of people will need it.

**Comment by ameya-deshmukh** [comment]
Lmao
I don't care about utility, I care about building dope shit.
